src/app/components/List/AuthorsList.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Html5Entities } from 'html-entities';
+import { Link } from 'react-router';
+
+const htmlEntities = new Html5Entities();
+
+const getIdentifier = author => (author.viaf && 'viaf') || (author.lcnaf && 'lcnaf') || 'name';
+
+const linkToAuthor = author => ({
+  q: author[getIdentifier(author)],
+  field: getIdentifier(author),
+  showQuery: `"${author.name}"`,
+  showField: 'author',
+});
+const birthDate = entity => (
+  <span>
+    {(entity.birth_date_display || entity.death_date_display) && <span> (</span>}
+    {entity.birth_date_display && <span>{entity.birth_date_display}</span>}
+    {entity.death_date_display && <span>{` --${entity.death_date_display}`}</span>}
+    {(entity.birth_date_display || entity.death_date_display) && <span>) </span>}
+  </span>
+);
+
+const AuthorsList = ({ agents }) => (
+  <ul className="authors-list">
```

Comment:
We may want to set up some naming conventions unless we have some already for CSS selectors. [projectAbbrev-classname] or [nypl-projectAbbrev-className] or [nypl-className].